pallets/identity/src/lib.rs

```diff
@@ -255,6 +261,31 @@ decl_module! {
             use polymesh_primitives::{identity_claim::IdentityClaimOld, migrate::migrate_map};
             migrate_map::<IdentityClaimOld>(b"Identity", b"Claims");
 
+            // Covert old scopes to new scopes
+            use frame_support::migration::{StorageIterator, put_storage_value};
+            let old_map = StorageIterator::<IdentityClaim>::new(b"Identity", b"Claims")
+                .drain().
+                collect::<Vec<(Vec<u8>, IdentityClaim)>>();
+
+            for (raw_key, value) in old_map.iter() {
+                let mut unhashed_key = Blake2_128Concat::reverse(&raw_key);
+                if let Ok(k1) = Claim1stKey::decode(&mut unhashed_key) {
+                    let mut raw_k2 = Blake2_128Concat::reverse(unhashed_key);
+                    if let Ok(k2) = Claim2ndKeyOld::decode(&mut raw_k2) {
+                        let k2 = Claim2ndKey {
+                            issuer: k2.issuer,
+                            scope: k2.scope.map(|did| Scope::Identity(did)),
+                        };
+                        let mut k1_hashed = k1.using_encoded(Blake2_128Concat::hash);
+                        let mut k2_hashed = k2.using_encoded(Blake2_128Concat::hash);
```

Comment:
```suggestion
                        k1_hashed.append(k2.using_encoded(Blake2_128Concat::hash));
```
Reduces 1 memcpy and simpler.